specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/RoleManagementPolicy.json

```diff
@@ -247,11 +250,14 @@
           "type": "boolean",
           "description": "The role management policy is default policy."
         },
-        "lastUpdatedDateTime": {
+        "lastModifiedBy": {
```

Comment:
these kind of properties must be enabled via the new ARM platform level capability called systemData. Please look up ARM wiki for systemData.

Reply:
We are dependent on PAS APIs for some of our data, and since they haven't implemented systemData we got an exception to not do it as well for now.